usaspending_api/agency/v2/views/federal_account_list.py

```diff
@@ -58,20 +68,42 @@ def format_results(self, rows):
     def get(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         self.sortable_columns = ["name", "obligated_amount", "gross_outlay_amount"]
         self.default_sort_column = "obligated_amount"
-        results = self.format_results(self.get_federal_account_list())
-        page_metadata = get_pagination_metadata(len(results), self.pagination.limit, self.pagination.page)
+        results = self.format_results(self.get_tbr_from_file_a_queryset(), self.get_federal_account_list_from_file_b())
+        page_metadata = get_pagination_metadata(len(results), len(results), self.pagination.page)
+        print(page_metadata)  # debug
```

Comment:
Should remove this `print()`.

Reply:
Removed print statement in commit 4f77912ecff45e5bb3ba1c9c4960e7bc575de59d